### DISCUSSION TITLE
UTF-8 statt ISO-8859-1

### DISCUSSION THREAD

**Discussion by dippeal** [openjverein/jverein#98] [Ideas]
Hallo Leute,
bevor ich mir den Aufwand mache die kompletten sourcen in UTF-8 zu konvertieren wollte ich fragen, wie ihr dazu steht.

**Comment by BT-11** [comment]
War nicht die Abhängigkeit zum Jamaica Framework der Grund warum ISO-8859-1 benutzt wurde? 
Generell würde ich UTF-8 aber bevorzugen.

**Reply by willuhn** [reply]
Jameica verwendet zwar ISO-8859-1. Die Plugins können aber problemlos auch UTF-8 verwenden.

**Comment by SchachtnerTh** [comment]
Keine Ahnung, ob ich mich als Neuling hier einmischen sollte...
Aber ich denke, dass das Projekt, wenn es neu erstellt werden würde, sicherlich UTF-8 als Zeichensatz verwenden würde.
Aber gibt es aktuell denn echte Einschränkungen durch ISO-8859-1?
Ich befürchte, dass wir durch diese Änderung einige Probleme einhandeln könnten, wenn bei der Konvertierung von Benutzerdaten in der Datenbank irgendwas schiefläuft...
(Ich gehe mal davon aus, dass die Daten in der Datenbank auch ISO-8859-1 sind und dann in Bestandsinstallationen auch konvertiert werden würden, oder? Oder würden nur die Sourcen konvertiert werden und die Datenbank unverändert bleiben?)

**Comment by dippeal** [comment]
Die Überlegung damals war, das die neuen IDE's standardmäßig mit UTF8 arbeiten und wenn jemand nicht weiß, dass alle OpenJVerein java Dateien mit ISO codiert sind knallt es beim PR.
